src/enveasy/parser.py

```diff
@@ -1,19 +1,21 @@
 import toml
 
 
-def parse_envs_from_toml(file_path):
+def parse_envs_from_toml(file_path, debug=False):
```

Comment:
Why we are using debug ??

Reply:
> Why we are using debug ??

it was for initial testing, removing it...